src/style.css

```
@@ -1 +1,72 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  max-width: 1200px;
```

Don't hardcode `max-width`. The header should be responsive on different screens sizes
```suggestion
  max-width: 1200px;
```